testshared/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     implementation Dependencies.androidx_recyclerview
     implementation Dependencies.kotlin_stdlib
     implementation Dependencies.androidx_core_ktx
+    implementation Dependencies.androidx_work_runtime
```

Comment:
I think this is only needed because of the unused `networkDeferred` overload.

Reply:
No, it is used in other places too.